README.md

```diff
@@ -205,7 +205,7 @@ Yes, ESLint natively supports parsing JSX syntax (this must be enabled in [confi
 
 ### What about ECMAScript 6 support?
 
-ESLint has full support for ECMAScript 6. By default, this support is off. You can enable ECMAScript 6 support through [configuration](http://eslint.org/docs/user-guide/configuring).
+You can separately enable ECMAScript 6 syntax and global variables through [configuration](http://eslint.org/docs/user-guide/configuring).
```

Comment:
Yikes, I also goofed on my comment - I copied and pasted wrong and didn't catch it. Sorry about that! 

I meant to say the following in my original comment:
> I think the word "separately" is confusing in this sentence because it's not entirely clear what is separate from what. Can we drop it so that the sentence is just "You can enable ECMAScript 6 syntax and global variables through configuration."?

So just dropping the word "separately". I think we should leave the first two sentences since they explain why the configuration is necessary. So the full thing would be:
`ESLint has full support for ECMAScript 6. By default, this support is off. You can enable ECMAScript 6 syntax and global variables through [configuration](http://eslint.org/docs/user-guide/configuring).`

Sorry again for the confusion - totally my fault!